__tests__/triggers/userCreateTrigger.ts

```diff
@@ -0,0 +1,27 @@
+import { DataSource } from 'typeorm';
+import createOrGetConnection from '../../src/db';
+import { usersFixture } from '../fixture/user';
+import { saveFixtures } from '../helpers';
+import { Alerts, User } from '../../src/entity';
+
+let con: DataSource;
+
+beforeAll(async () => {
+  con = await createOrGetConnection();
+});
+
+beforeEach(async () => {
+  jest.resetAllMocks();
+});
+
+describe('user_create_trigger after insert trigger', () => {
```

Comment:
I think we have already some tests on a user trigger, try to check where they are and move it to the right place but I'm not sure. Anyway not blocking

Reply:
I tried to search tests for triggers but only found one suite for the post keyword. I will move them together if I find the similar tests for triggers.

Reply:
Maybe a better place would be the users tests. For your consideration

Reply:
That makes sense. There are not a lot of triggers, hosting them together with other suites should not be confusing. Will move this one.